subcommands/keys/ca_revoke_device_ca.go

```diff
@@ -152,11 +152,11 @@ func doRevokeDeviceCa(cmd *cobra.Command, args []string) {
 	}
 	for serial := range toRevoke {
 		if _, ok := validSerials[serial]; !ok {
-			subcommands.DieNotNil(fmt.Errorf("There is no actual device CA with serial %s", serial))
+			subcommands.DieNotNil(fmt.Errorf("There is no device CA with serial %s", serial))
```

Comment:
Please, add `active device CA` or smth like that.
The thing is: there might exist a device CA with this serial, but it is not active in the user's factory.
E.g. for the "disable" command it might have been already disabled (and not yet revoked) or revoked earlier; for "revoke" command it might have been revoked earlier.